docs/gff3_sort.md

```diff
@@ -32,8 +32,6 @@ Sort features in a gff3 file by according to their order on a scaffold, their co
     - Sort multi-isoform gene models by feature type (default: False)
 6. -v, --version
     - show program's version number and exit
-7. -r, --reference
```

Comment:
I am not pretty sure if this should be removed.

Reply:
@ShangYuChiang

Reply:
The problem was resolved.